web/src/client/software.js

```diff
@@ -348,6 +246,73 @@ class ProductBaseClient {
       }
     });
   }
+
+  /**
+   * Returns the registration of the selected product.
+   *
+   * @return {Promise<Registration>}
+   */
+  async getRegistration() {
+    const response = await this.client.get("/software/registration");
+    if (!response.ok) {
+      console.log("Failed to get registration config: ", response);
```

Comment:
No need to add a whitespace there, `console.log` takes care.

```suggestion
      console.log("Failed to get registration config:", response);
```